pennylane/devices/default_qubit_jax.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains an jax implementation of the :class:`~.DefaultQubit`
+reference plugin.
+"""
+
+
+from pennylane.operation import DiagonalOperation
+
+from pennylane.devices import DefaultQubit
+from pennylane.devices import jax_ops
+import numpy as np
+
+try:
+    import jax.numpy as jnp
+    import jax
+
+except ImportError as e:
+    raise ImportError("default.qubit.jax device requires installing jax>0.2.0") from e

```

Comment:
I think it's fair to use `#pragma: no cover` here if you would like. pytest doesn't have a good way of mocking imports without monkeypatching `sys`, and even then it's really easy to introduce side effects that aren't torn down before other tests.